spec/erb_spec.rb

```diff
@@ -1,10 +1,11 @@
 require File.dirname(__FILE__) + '/spec_helper'
+require 'redcloth/erb_extension'
```

Comment:
Requiring this here requires it globally. Not just for this test. Better would be to create a test class and include it in there.